src/token/partial-soulbound/IPartialSoulboundToken.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
+
+import { IUUPS } from "../../lib/interfaces/IUUPS.sol";
+import { IERC721Votes } from "../../lib/interfaces/IERC721Votes.sol";
+import { IERC5192 } from "../../lib/interfaces/IERC5192.sol";
+import { IManager } from "../../manager/IManager.sol";
+import { IBaseToken } from "../interfaces/IBaseToken.sol";
+import { PartialSoulboundTokenTypesV1 } from "./types/PartialSoulboundTokenTypesV1.sol";
+
+/// @title IToken
+/// @author Neokry
+/// @notice The external Token events, errors and functions
+interface IPartialSoulboundToken is IUUPS, IERC721Votes, IBaseToken, IERC5192, PartialSoulboundTokenTypesV1 {
```

Comment:
nit: personally i prefer `account-` compared to `soul-` bound tokens, even though i know vitalik wrote about `soul-bound` they technically really are closer to `account-bound` and the terminology feels cleaner for me.